<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetValueDialog</class>
 <widget class="QDialog" name="SetValueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_rho">
       <property name="text">
        <string>Rho</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_maxrho">
       <property name="text">
        <string>maxRho</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_minrho">
       <property name="text">
        <string>minRho</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_minangle">
       <property name="text">
        <string>minAngle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_maxlength">
       <property name="text">
        <string>maxLength</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_rho"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_maxrho"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_minrho"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_minangle"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_maxlength"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_exit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_sure">
       <property name="text">
        <string>Sure</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
